wheaton-college-phd-in-biblical-and-theological-studies.csl

```diff
@@ -331,8 +333,10 @@
         <text term="in" text-case="capitalize-first" suffix=" "/>
         <choose>
           <if variable="volume" match="any">
-            <text term="volume" form="short" suffix=" "/>
-            <text variable="volume" prefix=" " suffix=" of "/>
+            <group delimiter=" ">
+              <text term="volume" form="short"/>
+              <text variable="volume" suffix=" of "/>
```

Comment:
I am no longer studying there, but based on the current style guide at http://www.wheaton.edu/~/media/Files/Academics/Departments/BTS/PhD%20Program/Style_Guide.pdf and my memory, the only place you include the number of volumes is when you are citing a work as a whole or in the bibliography. But just by looking at the commit I cannot tell if this creates a problem or not. Here is an example of how things should be: 

Note: G. E. Ladd, “Kingdom of God,” in vol 3 of *The International Standard Bible Encyclopedia*, ed. Geoffrey W. Bromiley, rev. ed. (Grand Rapids: Eerdmans, 1986), 29.
Bibliography: G. E. Ladd, “Kingdom of God.” In vol 3 of *The International Standard Bible Encyclopedia*, edited by Geoffrey W. Bromiley, 23–39. Rev. ed. Grand Rapids: Eerdmans, 1986. 

But this commit does not change that behavior, right?